src/components/SingleVideo/SingleVideo.jsx

```diff
@@ -0,0 +1,198 @@
+                    {video.description}
+                  </p>
+                  <span className="text-md margin-b text-bold">Comments</span>
+                  <div className="add-comment">
+                    <div
+                      className="avatar avatar-text-xs avatar-text img-round text-md text-bold margin-r"
+                      role="img"
+                      alt="Avatar"
+                    >
+                      MD
+                    </div>
+                    <input
+                      className="input standard"
+                      type="text"
+                      placeholder="Type your commentt"
+                    />
+                    <button className="btn btn-solid-primary margin-l">
+                      Comment
+                    </button>
+                  </div>
+                  <div className="show-comment">
+                    <div className="user-avatar margin-r">
+                      <div
+                        className="avatar avatar-text-xs avatar-text img-round text-md text-bold"
+                        role="img"
+                        alt="Avatar"
+                      >
+                        MD
+                      </div>
+                    </div>
+
+                    <div className="user-comment">
+                      <span className="text-md text-bold">Deekshith M D</span>
+                      <span className="comment-text text-md text-justify">
+                        This video gave complete information about farming
+                        techniques required to get succcess in this crop
+                      </span>
+                    </div>
+                  </div>
+                </div>
+              )
+            );
+          })}
+          <div className="suggested-list">
+            <p className="text-md text-bold">Recommended Videos...</p>
+            {data.videos.map((video) => {
+              return (
+                video._id !== videoId && (
+                  <div
+                    className="video-card"
+                    key={video._id}
+                    onClick={() => showSingleVideo(video)}
+                  >
+                    <div className="video-image">
+                      <img
+                        src={video.videoThumbnail}
+                        alt="thumb"
+                        className="img-responsive"
+                      />
+                    </div>
+                    <div className="video-details text-md text-bold">
+                      <div className="video-header">
+                        <span className=" video-title text-justify">
+                          {video.title}
+                        </span>
+                        <i className="fa-solid fa-ellipsis-vertical options"></i>
+                      </div>
+                      <div className="video-footer text-sm">
+                        <span>{video.creator}</span>
+                        <span>12/05/2018</span>
```

Comment:
I think you have written this hardcoded date by mistake.

Reply:
yes, thank you for pointing it out...